helm/drone-fork-approval-extension/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
+{{- if .Values.ingress.enabled }}
+{{- range $host := .Values.ingress.hosts }}
+  {{- range .paths }}
+  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ $host.host }}{{ .path }}
+  {{- end }}
+{{- end }}
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "drone-fork-approval-extension.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "drone-fork-approval-extension.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "drone-fork-approval-extension.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "drone-fork-approval-extension.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  export CONTAINER_PORT=$(kubectl get pod --namespace {{ .Release.Namespace }} $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:$CONTAINER_PORT
+{{- end }}
```

Comment:
All of this boilerplate is pretty irrelevant for this particular chart's use case, you might just want to delete it to avoid confusion.

Reply:
I reworked this pretty substantially in 3bfe78a.  Now it looks like:

```console
$ helm upgrade -n drone drone-fork-approval-extension ./helm/drone-fork-approval-extension -f values.yml
Release "drone-fork-approval-extension" has been upgraded. Happy Helming!
NAME: drone-fork-approval-extension
LAST DEPLOYED: Tue May 18 22:27:59 2021
NAMESPACE: drone
STATUS: deployed
REVISION: 2
TEST SUITE: None
NOTES:
1. Check that the pods are up and running:

$ kubectl logs -n drone -l 'app.kubernetes.io/name=drone-fork-approval-extension'
time="2021-05-15T04:58:14Z" level=info msg="server listening on address :3888"

2. Update your Drone deployment to call this extension.
   Add one of the following to the Drone helm values file,
   and upgrade the deployment:

env:
  DRONE_VALIDATE_PLUGIN_SECRET: <the secret you selected>
  DRONE_VALIDATE_PLUGIN_ENDPOINT: http://drone-fork-approval-extension:3888

OR

extraSecretNamesForEnvFrom:
  - drone-fork-approval-extension
podAnnotations:
  checksum/drone-fork-approval-extension: 1be6321a100c19e5aab64efc8c4ba3a66b6442c1f52112fc53b6309c6b54f6d4
```